blockchain/state_manager/src/utils.rs

```diff
@@ -70,15 +72,25 @@ where
 
         let info = mas.info(store)?;
 
-        let spt = RegisteredSealProof::from_sector_size(info.sector_size(), nv.into());
+        let sector_size: SectorSize = info.sector_size();
+
+        let spt = <forest_shim::sector::RegisteredSealProof as Inner>::FVM::from_sector_size(
```

Comment:
Let's add the `from_sector_size` function to the shimmed version of `RegisteredSealProof`

Reply:
that's an additional impl we'll need to keep in sync with upstream? still do it?

Reply:
No need to keep it in sync. It can just forward to to the real `from_sector_size` function.

Reply:
added